src/main/java/com/kainos/ea/controller/CapabilityController.java

```diff
@@ -0,0 +1,43 @@
+package com.kainos.ea.controller;
+import com.kainos.ea.dao.CapabilitiesDao;
+import com.kainos.ea.exception.DatabaseConnectionException;
+import com.kainos.ea.service.CapabilitiesService;
+import com.kainos.ea.util.DatabaseConnector;
+import io.swagger.annotations.Api;
+import org.eclipse.jetty.http.HttpStatus;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("API Hack Street Boys")
+@Path("/api")
+
+public class CapabilityController {
+
+    private CapabilitiesService capabilitiesService;
+
+    public CapabilityController()
+    {
+        capabilitiesService = new CapabilitiesService(new CapabilitiesDao(), new DatabaseConnector());
+    }
+
+    public CapabilityController(CapabilitiesService capabilitiesService) {this.capabilitiesService = capabilitiesService;}
+
+    @GET
+    @Path("/capabilities")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCapabilities()
+    {
+        try
+        {
+            return Response.ok(capabilitiesService.getCapabilities()).build();
+        } catch (SQLException | DatabaseConnectionException e)
```

Comment:
Could we create a custom exception that we can use for this scenario? 

```suggestion
        } catch (FailedToGetCapabilityException e)
```